include/boost/gil/io/path_spec.hpp

```diff
@@ -1,5 +1,6 @@
 //
 // Copyright 2007-2008 Andreas Pokorny, Christian Henning
+// Copyright 2024 Dirk Stolle (minor fixes)
```

Comment:
@striezel IMO  `(minor fixes)` note is not necessary

Reply:
Alright. The note has been removed in commit a27fc03b526d2c45c337815f40f9f1001490657b.